TaskLayer/XLSearchTask/XLSearchTask.cs

```diff
@@ -145,7 +145,23 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
                 filteredAllPsms.Add(RemoveDuplicateFromPsmsPerScan(psmsPerScan));
             }
 
-            var allPsmsList = filteredAllPsms.Select(p => p.First()).OrderByDescending(p => p.XLTotalScore).ToList();
+            var allPsmsList = new List<CrosslinkSpectralMatch>();
```

Comment:
can you add a comment explaining the `List<List<CrosslinkSpectralMatch>>`? I don't understand why there is a list of lists

Reply:
Comment added.